org.altaqwaa.Altaqwaa.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Altaqwaa
+Exec=zypak-wrapper.sh /app/bin/altaqwaa "$@"
+Terminal=false
+Type=Application
+Icon=org.altaqwaa.rn0x
```

Comment:
why not naming the icon correctly ?

Reply:
So... I've already raised this to upstream https://github.com/rn0x/Altaqwaa-Islamic-Desktop-Application/pull/26

For now, this is just for compatibility.

Reply:
I don't see what's the problem is for "compatibility". The icon has to match the app-id.

Reply:
Oh, you're right. I just copied them from the ones in the flatpak electron-builder makes. I'll make one 256x256 icon named "org.altaqwaa.Altaqwaa" instead.